src/routes/Strategies/Pending.js

```diff
@@ -1,7 +1,106 @@
-import React from "react";
+import React, { useEffect, useContext, useCallback, useState } from "react";
 
+import findPendingStrategiesForOwner from "api/safe/findPendingStrategiesForOwner";
+
+import { Web3Context } from "components/Web3Provider";
+
+import { Loader } from "@gnosis.pm/safe-react-components";
+
+import {
+  Table,
+  TableHead,
+  TableRow,
+  TableCell,
+  TableBody,
+  TableContainer,
+} from "@material-ui/core";
+
+import Decimal from "decimal.js";
 const Pending = () => {
-  return <div>Pending strategies</div>;
+  const context = useContext(Web3Context);
+  const [strategies, setStrategies] = useState(null);
+  const handleLoadPending = useCallback(async () => {
+    const pendingStrategies = await findPendingStrategiesForOwner(context);
+    setStrategies(pendingStrategies);
+    console.log(pendingStrategies);
+  }, [context]);
+
+  useEffect(() => {
+    handleLoadPending();
+  }, []);
+
+  if (!strategies) {
+    return (
+      <div>
+        <Loader size="lg" />
+      </div>
+    );
+  }
+
+  return (
+    <TableContainer>
+      <Table>
+        <TableHead>
+          <TableRow>
+            <TableCell>Created</TableCell>
+            <TableCell>Token Pair</TableCell>
+            <TableCell>Nonce</TableCell>
+            <TableCell>Price Range</TableCell>
+            <TableCell>Brackets</TableCell>
+            <TableCell>Status</TableCell>
+            <TableCell />
+          </TableRow>
+        </TableHead>
+        <TableBody>
+          {strategies.map((strategy) => {
+            return (
+              <TableRow key={strategy.transactionHash}>
+                <TableCell>{strategy.created.toLocaleString()}</TableCell>
+                <TableCell>
+                  {strategy.quoteTokenDetails?.symbol} -{" "}
+                  {strategy.baseTokenDetails?.symbol}
+                </TableCell>
+                <TableCell>{strategy.nonce}</TableCell>
+                <TableCell>
+                  {strategy.prices[0]
+                    .div(
+                      new Decimal(
+                        Math.pow(
+                          10,
+                          strategy.quoteTokenDetails.decimals -
+                            strategy.baseTokenDetails.decimals
+                        )
+                      )
+                    )
+                    // .div(Math.pow(10, strategy.quoteTokenDetails.decimals))
+                    .toSD(4)
+                    .toString()}
```

Comment:
Take a look at https://github.com/gnosis/dex-js/blob/96ccf8ea4acc9420cfe384f7bcc4dd8eaaf84036/src/utils/format.ts#L197, https://github.com/gnosis/dex-js/blob/96ccf8ea4acc9420cfe384f7bcc4dd8eaaf84036/src/utils/format.ts#L140 and https://github.com/gnosis/dex-js/blob/96ccf8ea4acc9420cfe384f7bcc4dd8eaaf84036/src/utils/format.ts#L350.
Those functions use BN and BigNumber, but nothing stops us from adding support for Decimal as well.
Or even convert it to one of those before calling the functions as they are.

Reply:
Not super sure if this needs to be moved to BN? I think Decimal.js is sufficient. It is a dependency that could potentially be removed at some point though when changing to this approach.

Reply:
My point is just that there are utils for doing the price/value calculation in place to make our life easier :)